source/acquaman/BioXAS/BioXASGenericStepScanConfiguration.cpp

```diff
@@ -32,3 +34,63 @@ AMScanController* BioXASGenericStepScanConfiguration::createController()
 
 	return controller;
 }
+
+bool BioXASGenericStepScanConfiguration::usingAnyGeDetector() const
```

Comment:
I remembered these functions were living in the BioXASBeamlineSupport.h. The purpose of those functions are to check whether a given scan configuration is using a detector/control/scaler or not. Technically, the scan configuration knows whether they are using what. Therefore, this should be better to be a property of the scan configuration class instead of a static function.